install/kubernetes/cilium/templates/hubble-relay-service.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.hubble.relay.enabled }}
+kind: Service
+apiVersion: v1
+metadata:
+  name: hubble-relay
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: hubble-relay
+spec:
+  type: ClusterIP
+  selector:
+    k8s-app: hubble-relay
+  ports:
+  - protocol: TCP
+{{- if .Values.hubble.relay.servicePort }}
+    port: {{ .Values.hubble.relay.servicePort }}
+{{- else if .Values.hubble.relay.tls.enabled }}
```

Comment:
.Values.hubble.relay.tls.enabled -> .Values.hubble.relay.tls.server.enabled

Reply:
:thinking: I wonder if we could write a script to generate all of the correct setting options, then grep across `install/kubernetes/*` for any `.Values` options that are not declared in the values file and flag a warning / add check to CI that you're probably defining an option that is unused?